scss/_variables.scss

```diff
@@ -1041,8 +1041,8 @@ $form-check-input-checked-bg-image:       var(--#{$boosted-prefix}check-icon) !d
 $form-check-input-disabled-color:         $gray-900 !default; // Boosted mod
 $form-check-radio-checked-bg-image:       url("data:image/svg+xml,<svg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'><circle r='2' fill='#{$form-check-input-checked-color}'/></svg>") !default;
 
-$form-check-input-indeterminate-color:          color-contrast($component-active-color) !default;
-$form-check-input-indeterminate-bg-color:       $component-active-color !default;
+$form-check-input-indeterminate-color:          color-contrast($brand-orange) !default;
```

Comment:
So that indeterminate style is always linked and consistent to the checked color, can't we directly do?
```scss
$form-check-input-indeterminate-color:          $form-check-input-checked-color !default;
$form-check-input-indeterminate-bg-color:       $form-check-input-checked-bg-color !default;
```